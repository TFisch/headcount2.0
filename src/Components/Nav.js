import React from 'react';
import "../styling/Nav.css"
import Search from '../Components/Search'
import PropTypes from 'prop-types';
import '../images/grad.svg';

const Nav = ({ updateFilter }) => {
  return <div className="nav">
    <div className="left-wrapper">
      <img className="logo-icon" src={require('../images/grad.svg')} />
      <h3 className="logo">HeadCount 2.0</h3>
    </div>
    <Search updateFilter={updateFilter} />
  </div>
}

export default Nav;

Nav.propTypes = {
  updateFilter: PropTypes.func,
}