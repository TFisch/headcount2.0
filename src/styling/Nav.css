.logo {
  font-size: 2em;
  margin-left: 2em;
}

.nav {
  align-items: center;
  background-color: #05386B;
  border-bottom: 4px solid rgba(237, 245, 225, 0.829);
  display: flex;
  color: rgb(46, 182, 107);;
  font-family: 'Cabin', sans-serif;
  justify-content: space-between;
  height: 5em;
  text-shadow: 1px 1px rgba(237, 245, 225, 0.829);
  width: 100%;
}
