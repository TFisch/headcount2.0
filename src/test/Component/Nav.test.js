import React from 'react';
import { shallow, mount } from 'enzyme';
import Nav from '../../Components/Nav';

describe('Nav', () => {

  it('should match the snapshot', () => {
    let wrapper = shallow(<Nav />);


    expect(wrapper).toMatchSnapshot();
  });
});