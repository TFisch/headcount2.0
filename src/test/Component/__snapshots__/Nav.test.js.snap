// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="left-wrapper"
>
          <img
                    className="logo-icon"
                    src="grad.svg"
          />
          <h3
                    className="logo"
          >
                    HeadCount 2.0
          </h3>
</div>,
        <h4
          className="instruct-text"
>
          Select From the List Below or Search For a Specific District
</h4>,
        <Search
          updateFilter={undefined}
/>,
      ],
      "className": "nav",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              className="logo-icon"
              src="grad.svg"
/>,
            <h3
              className="logo"
>
              HeadCount 2.0
</h3>,
          ],
          "className": "left-wrapper",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "logo-icon",
              "src": "grad.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "HeadCount 2.0",
              "className": "logo",
            },
            "ref": null,
            "rendered": "HeadCount 2.0",
            "type": "h3",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Select From the List Below or Search For a Specific District",
          "className": "instruct-text",
        },
        "ref": null,
        "rendered": "Select From the List Below or Search For a Specific District",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "updateFilter": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="left-wrapper"
>
            <img
                        className="logo-icon"
                        src="grad.svg"
            />
            <h3
                        className="logo"
            >
                        HeadCount 2.0
            </h3>
</div>,
          <h4
            className="instruct-text"
>
            Select From the List Below or Search For a Specific District
</h4>,
          <Search
            updateFilter={undefined}
/>,
        ],
        "className": "nav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                className="logo-icon"
                src="grad.svg"
/>,
              <h3
                className="logo"
>
                HeadCount 2.0
</h3>,
            ],
            "className": "left-wrapper",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "logo-icon",
                "src": "grad.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "HeadCount 2.0",
                "className": "logo",
              },
              "ref": null,
              "rendered": "HeadCount 2.0",
              "type": "h3",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Select From the List Below or Search For a Specific District",
            "className": "instruct-text",
          },
          "ref": null,
          "rendered": "Select From the List Below or Search For a Specific District",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "updateFilter": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "prevContext": true,
          },
        },
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
